src/screens/general/Investment.js

```diff
@@ -1,17 +1,83 @@
 import React from 'react';
+import 'react-circular-progressbar/dist/styles.css';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import { connect } from 'react-redux';
 import '../../styles/Investments.css';
+import GreenCheck from '../../assets/green_check.png';
+import RedX from '../../assets/red_x.png';
 
 class Investment extends React.PureComponent {
   render() {
+    const { owner } = this.props;
+    const percentage = owner.numberOfShares * 10;
     return (
       <div className="dashboard">
         <div className="mainheader">
           <h1>My Investment</h1>
           <div className="columnformat">
             <div className="left-content">
               <h2>My Investment</h2>
-              <div className="investment-box-1">investments box 1 test</div>
-              <div className="investment-box-2">investments box 2 test</div>
+              <div className="investment-box-1">
+                <div className="circle-progress-bar">
+                  <CircularProgressbar
+                    viewBox="0 0 0 0"
+                    value={percentage}
+                    text={owner.numberOfShares}
+                    styles={buildStyles({
+                      pathColor: '#cd6795',
+                      textSize: '30px',
+                      textColor: '#cd6795',
+                      trailColor: '#F4F1F24'
+                    })}
```

Comment:
```suggestion
                    styles={buildStyles(circularProgressBarStyles)}
```

Then, define (at the top of this file):

```
const circularProgressBarStyles = {
                      pathColor: '#cd6795',
                      textSize: '30px',
                      textColor: '#cd6795',
                      trailColor: '#F4F1F24'
                    }
```